### DISCUSSION TITLE
Use pip cache while offline?

### DISCUSSION THREAD

**Discussion by drui9** [python-poetry/poetry#5751] [Ideas]
Hello. I don't know if this has been requested before, but I need it bad. There's no reason to fail installing Flask, if the pip-cache already has a recently installed version. I've seen that pip has a download packages option. Doing this, then installing could leave a window for installation even when offline. Please consider this request

**Comment by fredrikaverpil** [comment]
Isn't this already supported?

Poetry has a `cache-dir` option, and [here](https://python-poetry.org/docs/configuration/#cache-dir) are the docs.

EDIT: Aah, I see, you want poetry to re-use pip's cache.
